### PR TITLE
fix: named apis and id configurations

### PR DESCRIPTION
**Pull request by odinuv**
except for `outputBucket` property, the output bucket name can also be generated by from id of the configuration or name of the api:
https://github.com/keboola/generic-extractor/blob/master/src/Executor.php#L81-L88

https://github.com/keboola/generic-extractor/blob/master/src/Configuration/Api.php#L85

https://github.com/keboola/generic-extractor/blob/master/src/Configuration/Api.php#L26

3.6.0 success job https://connection.keboola.com/admin/projects/578/jobs/585485812?q=
3.7.2 failed job https://connection.keboola.com/admin/projects/578/jobs/585657670?q=
PR#116 success job https://connection.keboola.com/admin/projects/578/jobs/585651784?q=

### CODE REVIEW

File: src/MissingTableHelper.php [hunk 1]
```diff
@@ -3,17 +3,28 @@
...
-    public static function checkConfigs($configs, $dataDir)
+    public static function checkConfigs($configs, $dataDir, Extractor $configuration)
     {
         foreach ($configs as $config) {
+            $api = $configuration->getApi($config->getAttributes());
+
+            if (!empty($config->getAttribute('outputBucket'))) {
+                $outputBucket = $config->getAttribute('outputBucket');
+            } elseif ($config->getAttribute('id')) {
+                $outputBucket = 'ex-api-' . $api->getName() . "-" . $config->getAttribute('id');
```
Comment (MiroCillik) [inline src/MissingTableHelper.php:19]: A v pripade, ze API `name` je empty? Tak ten $outputBucket bude mat tvar `ex-api-generic-configId`? Name bude `generic`?

Comment (MiroCillik) [reply src/MissingTableHelper.php:19]: Uz to vidim https://github.com/keboola/generic-extractor/blob/master/src/Configuration/Api.php#L26

Comment (MiroCillik) [reply src/MissingTableHelper.php:19]: Jasne, musi to byt stejne ako tu:
https://github.com/keboola/generic-extractor/blob/master/src/Executor.php#L85

Comment (MiroCillik) [reply src/MissingTableHelper.php:19]: A co ten `__kbc_default` teda? Aha, to sa tu zas vynulluje:
https://github.com/keboola/generic-extractor/blob/master/src/Executor.php#L120
Hh, aby to nebolo take jednoduche.

Comment (MiroCillik) [reply src/MissingTableHelper.php:19]: Jo ty si to vlastne nalinkoval v tej description :D

Comment (odinuv) [reply src/MissingTableHelper.php:19]: ja se ti snazim ulehcit pracu a ty si to ani neprectes :D

Comment (MiroCillik) [reply src/MissingTableHelper.php:19]: Nechcel som sa nechat ovplyvnit :D Takto sme obaja dosli k rovnakemu zaveru haha

**Pull request merged**